src/Service.php

```diff
@@ -28,6 +28,8 @@ public function __construct($serviceName)
      */
     public static function getService($serviceName)
     {
+        $serviceName = strtolower(preg_replace('/([A-Z])/', '-$1', $serviceName));
```

Comment:
@dsposito - That assumes single name services, which are always the case presently, but might not be in the future.